client/apps.go

```diff
@@ -261,3 +268,103 @@ func (client *Client) Restart(names []string, opts RestartOptions) (changeID str
 	}
 	return client.doAsync("POST", "/v2/apps", nil, nil, bytes.NewReader(buf))
 }
+
+func (app *AppInfo) Notes() string {
```

Comment:
I have a mild preference for this not to be a method but a function in some other package?  cmd?  client/clientutil?

Reply:
what's the reasoning for that?
No strong feelings about it, if there's a rationale for what goes where.

Reply:
We don't have any Notes methods in client so far, and sensibly afaict. Notes as a concept belong to the CLI UX, from that POV even clientutil seems the wrong place. It's not clear that clients (vs CLI UX) need notes or would agree on what they should be. Basically if client is about talking the snapd API from a client-side POV and considering we had requests to evolve client as a library vs just a piece of our clients/the CLI experience, it seems over-deterministic to put Notes under client/.

Reply:
From another point of view, it's a very Smalltalk impulse to attach methods to things because they exist, in Smalltalk everything is an object with methods (all in one program/memory image), Go has standalone functions as well tough, afair serious Smalltalk vendors spent a quite a few brain cycles working on layered concepts to disentangle/manage the consequences of that impulse in the end.